### PR TITLE
keepassx2: update to 2.0-beta1

### PR DESCRIPTION
**Pull request by Grauwolf**

### CODE REVIEW

**Comment by Gottox** [comment]
:+1:

**Comment by xtraeme** [comment]
Please fix the build with `musl`, this file `/builddir/keepassx-2.0-beta1/src/core/qlockfile_unix.cpp` needs `<sys/fcntl.h>` for the missing defs.

Thanks.

**Comment by Grauwolf** [comment]
Should be good now.

File: srcpkgs/keepassx2/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
 # Template file for 'keepassx2'
 _realname="keepassx"
-_realversion="2.0-alpha6"
+_realversion="2.0-beta1"
 pkgname="${_realname}2"
 version="2.0"
-revision=1
+revision=3
```
Comment (beefcurtains) [inline srcpkgs/keepassx2/template]: 3 is the new 2

**Comment by xtraeme** [comment]
For updates and new packages `revision` must be reset to `1`.

**Comment by beefcurtains** [comment]
version isn't incremented

**Comment by xtraeme** [comment]
It it, updated from alpha to beta.

**Comment by beefcurtains** [comment]
It says ``version="2.0"`` :grey_question:

**Comment by xtraeme** [comment]
Hmm you are right, I thought `${_realversion}` was used.

**Pull request merged**